httpstan/cache.py

```diff
@@ -94,12 +104,10 @@ def dump_fit(fit_bytes: bytes, name: str) -> None:
         name: Stan fit name
         fit_bytes: LZ4-compressed messages associated with Stan fit.
     """
-    cache_path = appdirs.user_cache_dir("httpstan", version=httpstan.__version__)
     # fits are stored under their "parent" models
-    fits_path = os.path.join(*([cache_path] + name.split("/")[:-1]))
-    fit_filename = os.path.join(fits_path, f'{name.split("/")[-1]}.jsonlines.lz4')
-    os.makedirs(fits_path, exist_ok=True)
-    with open(fit_filename, mode="wb") as fh:
+    fit_filename = fit_filepath(name)
```

Comment:
This should be `fit_path` not `fit_filename`, right?

Reply:
Yes, fit_path sounds better